docs/release-notes/flink-1.9.md

```diff
@@ -0,0 +1,239 @@
+
+Flink 1.9.0 provides support for two planners for the Table API, namely Flink's original planner and the new Blink
+planner. The original planner maintains same behaviour as previous releases, while the new Blink planner is still
+considered experimental and has the following limitations:
+
+- The Blink planner can not be used with `BatchTableEnvironment`, and therefore Table programs ran with the planner can not 
+be transformed to `DataSet` programs. This is by design and will also not be supported in the future. Therefore, if
+you want to run a batch job with the Blink planner, please use the new `TableEnvironment`. For streaming jobs,
+both `StreamTableEnvironment` and `TableEnvironment` works.
+- Implementations of `StreamTableSink` should implement the `consumeDataStream` method instead of `emitDataStream`
+if it is used with the Blink planner. Both methods work with the original planner.
+This is by design to make the returned `DataStreamSink` accessible for the planner.
+- Due to a bug with how transformations are not being cleared on execution, `TableEnvironment` instances should not
+be reused across multiple SQL statements when using the Blink planner.
+- `Table.flatAggregate` is not supported
+- Session and count windows are not supported when running batch jobs.
+- The Blink planner only supports the new `Catalog` API, and does not support `ExternalCatalog` which is now deprecated.
+
+Related issues:
+- [FLINK-13708: Transformations should be cleared because a table environment could execute multiple job](https://issues.apache.org/jira/browse/FLINK-13708)
+- [FLINK-13473: Add GroupWindowed FlatAggregate support to stream Table API (Blink planner), i.e, align with Flink planner](https://issues.apache.org/jira/browse/FLINK-13473)
+- [FLINK-13735: Support session window with Blink planner in batch mode](https://issues.apache.org/jira/browse/FLINK-13735)
+- [FLINK-13736: Support count window with Blink planner in batch mode](https://issues.apache.org/jira/browse/FLINK-13736)
+
+### SQL DDL
+
+In Flink 1.9.0, the community also added a preview feature about SQL DDL, but only for batch style DDLs.
+Therefore, all streaming related concepts are not supported yet, for example watermarks.
+
+Related issues:
+- [FLINK-13661: Add a stream specific CREATE TABLE SQL DDL](https://issues.apache.org/jira/browse/FLINK-13661)
+- [FLINK-13568: DDL create table doesn't allow STRING data type](https://issues.apache.org/jira/browse/FLINK-13568)
+
+### Java 9 support
+
+Since Flink 1.9.0, Flink can now be compiled and run on Java 9. Note that certain components interacting
+with external systems (connectors, filesystems, metric reporters, etc.) may not work since the respective projects may
+have skipped Java 9 support.
+
+Related issues:
+- [FLINK-8033: JDK 9 support](https://issues.apache.org/jira/browse/FLINK-8033)
+
+## Deprecations and breaking changes
+
+### Scala expression DSL for Table API moved to `flink-table-api-scala`
```

Comment:
@twalthr Could you take a look at this latest addition?

Reply:
Looks good to me. Thanks for the update.